lighthouse-core/audits/byte-efficiency/uses-optimized-images.js

```diff
@@ -93,65 +80,56 @@ class UsesOptimizedImages extends Audit {
         return results;
       }
 
-      const originalKb = Math.round(image.originalSize / KB_IN_BYTES);
+      const url = URL.getDisplayName(image.url);
       const webpSavings = UsesOptimizedImages.computeSavings(image, 'webp');
 
       if (webpSavings.bytes > WEBP_ALREADY_OPTIMIZED_THRESHOLD_IN_BYTES) {
         hasAllEfficientImages = false;
+      } else if (webpSavings.bytes < IGNORE_THRESHOLD_IN_BYTES) {
+        return results;
       }
 
       let jpegSavingsLabel;
       if (/(jpeg|bmp)/.test(image.mimeType)) {
         const jpegSavings = UsesOptimizedImages.computeSavings(image, 'jpeg');
-        if (jpegSavings.bytes > 0) {
+        if (jpegSavings.bytes > JPEG_ALREADY_OPTIMIZED_THRESHOLD_IN_BYTES) {
           hasAllEfficientImages = false;
           jpegSavingsLabel = `${jpegSavings.percent}%`;
         }
       }
 
       totalWastedBytes += webpSavings.bytes;
+
       results.push({
-        url: URL.getDisplayName(image.url),
+        url,
         preview: {url: image.url, mimeType: image.mimeType},
-        total: `${originalKb} KB`,
+        totalBytes: image.originalSize,
+        wastedBytes: webpSavings.bytes,
         webpSavings: `${webpSavings.percent}%`,
         jpegSavings: jpegSavingsLabel
       });
       return results;
     }, []);
 
-    let displayValue = '';
-    if (totalWastedBytes > 1000) {
-      const totalWastedKb = Math.round(totalWastedBytes / KB_IN_BYTES);
-      // Only round to nearest 10ms since we're relatively hand-wavy
-      const totalWastedMs = Math.round(totalWastedBytes / networkThroughput * 100) * 10;
-      displayValue = `${totalWastedKb}KB (~${totalWastedMs}ms) potential savings`;
-    }
-
     let debugString;
     if (failedImages.length) {
       const urls = failedImages.map(image => URL.getDisplayName(image.url));
       debugString = `Lighthouse was unable to decode some of your images: ${urls.join(', ')}`;
     }
 
-    return UsesOptimizedImages.generateAuditResult({
-      displayValue,
+    return {
+      passes: hasAllEfficientImages && totalWastedBytes < TOTAL_WASTED_BYTES_THRESHOLD,
       debugString,
-      rawValue: hasAllEfficientImages && totalWastedBytes < TOTAL_WASTED_BYTES_THRESHOLD,
-      extendedInfo: {
-        formatter: Formatter.SUPPORTED_FORMATS.TABLE,
-        value: {
-          results,
-          tableHeadings: {
-            preview: '',
-            url: 'URL',
-            total: 'Original (KB)',
-            webpSavings: 'WebP Savings (%)',
-            jpegSavings: 'JPEG Savings (%)',
-          }
-        }
+      results,
+      tableHeadings: {
+        preview: '',
+        url: 'URL',
+        totalKb: 'Original (KB)',
+        wastedKb: 'Savings (KB)',
```

Comment:
just a thought: should we put total savings last...as a way to indicate it's representative of the webp/jpeg column that provides the most impact.